base/Dockerfile

```diff
@@ -43,7 +43,7 @@ RUN useradd -l -u 33333 -G sudo -md /home/gitpod -s /bin/bash -p gitpod gitpod \
 ENV HOME=/home/gitpod
 WORKDIR $HOME
 # custom Bash prompt
-RUN { echo && echo "PS1='\[\e]0;\u \w\a\]\[\033[01;32m\]\u\[\033[00m\] \[\033[01;34m\]\w\[\033[00m\] \\\$ '" ; } >> .bashrc
+RUN { echo && echo "PS1='\[\033[01;32m\]\u\[\033[00m\] \[\033[01;34m\]\w\[\033[00m\]$(__git_ps1 " (%s)") $ '" ; } >> .bashrc
```

Comment:
I'd like to share two thoughts here:

1. I notice that you are removing the `\[\e]0;\u \w\a\]` part of the `PS1` prompt. I initially had no idea what this was doing (and when testing it in Gitpod, it seemed to have no effect), but after a while I found out that this apparently sets the Terminal's _title bar_ ([source](https://unix.stackexchange.com/a/306721)). I think setting the Terminal title that way used to work in Gitpod's previous IDE (Theia), but it no longer works in the new IDE (VS Code), so it should be safe to remove. 👍 

2. While testing the proposed new syntax, the `$(__git_ps1 " (%s)")` part gave me surprising results: The branch name initially seemed correct, but it didn't update when changing directories or when switching to different Git branches (in the screenshot below, I've copy/pasted the above `RUN` line into the left Terminal, and then opened a new Terminal on the right in order to test the prompt -- as you'll notice, the branch name always stayed the same, even outside the Git repository or on a different branch):

<img width="1440" alt="Screenshot 2021-07-19 at 13 30 37" src="https://user-images.githubusercontent.com/599268/126159449-04a2b895-fa6b-4711-afad-364c24bffb28.png">

I believe this is because the special `$` character needs additional "escaping" here (i.e. it needs some extra `\` character(s) in front of it).

Since it's quite awkward and difficult to write a correct PS1 syntax in this situation (i.e. in a Dockerfile, that prints a command line into a `.bashrc` script file, that then gets interpreted by IDE Terminals 🤯), I believe the correctly escaped syntax would be `\$(__git_ps1 \" (%s)\")`, i.e. like so:

```suggestion
RUN { echo && echo "PS1='\[\033[01;32m\]\u\[\033[00m\] \[\033[01;34m\]\w\[\033[00m\]\$(__git_ps1 \" (%s)\") $ '" ; } >> .bashrc
```